mrp_operations_schedule/static/description/index.html

```diff
@@ -0,0 +1,83 @@
+<section class="oe_container">
```

Comment:
why is this file necessary ? it seems to contain the same information as readme.rst.
Regards.